cli/container.go

```diff
@@ -96,7 +107,7 @@ func createServer(verbose bool, image string, apiPort string, args []string, env
 	}
 
 	if len(volumes) > 0 && volumes[0] != "" {
```

Comment:
With the default volume set, this would be always true

Reply:
I think I will remove the `""` check, but I think it might be safest to keep the `> 0` check in case `checkServer` is ever being used without it in the future. Thoughts?

Reply:
This is reflected in HEAD. Can rescope it if it's preferred!